CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - On Android, calling `WindowEvent::Focused` now works properly instead of always returning false. 
+- On Windows, add `WindowBuilderExtWindows::with_theme(Theme::)` to set a preferred theme.
```

Comment:
Should append into the changelog, and not insert between. Also, there's no need to provide argument in changelog, just a name.

```suggestion
- On Windows, add `WindowBuilderExtWindows::with_theme` to set a preferred theme.
```

Reply:
👍 commited